app/api/domains/route.ts

```diff
@@ -64,25 +57,31 @@ setInterval(() => {
   currentDomainNameAvailabilityCheck = checkDomainNamesAvailability();
 }, 1000);
 
-export async function POST(request: Request) {
+export async function POST(request: NextRequest) {
```

Comment:
Are POST requests supported at the Edge?

Reply:
Yes, you can use any of the HTTP verbs: https://beta.nextjs.org/docs/routing/route-handlers